chutney/server/src/main/java/com/chutneytesting/campaign/infra/SchedulingCampaignDto.java

```diff
@@ -7,36 +7,64 @@
 
 package com.chutneytesting.campaign.infra;
 
+import static java.util.Collections.emptyList;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import java.time.LocalDateTime;
+import java.util.Collections;
 import java.util.List;
 
+@JsonPropertyOrder({ "id", "schedulingDate", "environment", "campaignsId", "campaignsTitle", "datasetsId" })
```

Comment:
Is it only to have clean json ? i don't see the use of this annotation really.

Reply:
yes it's only to fix the order in the json file